### ISSUE TITLE
Investigate UVW discrepancy between MS scripts and aavs_uv

### ISSUE THREAD

**Issue by telegraphic** [ska-sci-ops/aavs_uv#13]
Shinichiro reported a discrepancy between UVW coordinates as computed by the `aavs_uv` command-line tool, and the files produced by MS conversion script pipeline. 

I think the main issue was that MS pipeline had phased to the sun, whereas the `aavs_uv` tool had not. I have added a command-line flag, `-s`, to expose this functionality:

```
(aa3) IT1000016:tests daniel.price$ aavs_uv test-data/aavs2_2x500ms/correlation_burst_204_20230927_35116_0.hdf5 test.uvfits -o uvfits -s -n aavs3
2023-11-29 14:36:44.706 | INFO     | aavs_uv.converter:run:45 - aavs_uv 1.0.0
2023-11-29 14:36:44.706 | INFO     | aavs_uv.converter:run:48 - Telescope name: aavs3
2023-11-29 14:36:44.706 | INFO     | aavs_uv.converter:run:76 - Input path:    test-data/aavs2_2x500ms/correlation_burst_204_20230927_35116_0.hdf5
2023-11-29 14:36:44.706 | INFO     | aavs_uv.converter:run:77 - Array config:  /Users/daniel.price/Data/aavs_uv/src/aavs_uv/config/aavs3/uv_config.yaml
2023-11-29 14:36:44.706 | INFO     | aavs_uv.converter:run:78 - Output path:   test.uvfits
2023-11-29 14:36:44.706 | INFO     | aavs_uv.converter:run:79 - Output format: uvfits

2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:83 - Data shape:     (2, 1, 32896, 4)
2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:84 - Data dims:      ('time', 'frequency', 'baseline', 'polarization')
2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:85 - UTC start:      2023-09-27 09:45:16.013
2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:86 - MJD start:      60214.40643533284
2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:87 - LST start:      17.92774
2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:88 - Frequency 0:    159375000.0 Hz
2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:89 - Polarization:   ['XX' 'XY' 'YX' 'YY']

2023-11-29 14:36:45.032 | INFO     | aavs_uv.converter:run:95 - Loading test-data/aavs2_2x500ms/correlation_burst_204_20230927_35116_0.hdf5
2023-11-29 14:36:45.113 | INFO     | aavs_uv.converter:run:99 - Phasing to sun
2023-11-29 14:36:45.141 | INFO     | aavs_uv.converter:run:114 - Creating uvfits file: test.uvfits
Found antenna numbers > 255 in this data set. This is permitted by UVFITS standards, but may cause the `importuvfits` utility within CASA to crash. If attempting to use this data set in CASA, consider using the measurement set writer method (`write_ms`) instead.
2023-11-29 14:36:45.232 | INFO     | aavs_uv.converter:run:126 - Done. Time taken: Read: 0.11 s Write: 0.09 s Total: 0.20 s
```

### Comparing UVFITS files

To compare MS-generated UVFITS file against aavs_uv, I downloaded the file `chan_204_20231120T053539.uvfits` from `nimbus5:/data_archive/aavs3/real_time_calibration/2023_11_20-05:28` and the original HDF5 data, `correlation_burst_20_20231120_19774_0.hdf5`. I then used aavs_uv to create a comparison UVFITS file, with phase_to_sun applied.

In the FITS header, the MS file has a phase center:
```
CTYPE5  = 'RA      '                                                            
CRVAL5  =             235.0127                                                  
CRPIX5  =                   1.                                                  
CDELT5  =                   1.                                                  
CTYPE6  = 'DEC     '                                                            
CRVAL6  =            -19.55227
```

which is close (but not identical) to the aavs_uv version:

```
CTYPE6  = 'RA      '                                                            
CRVAL6  =   235.00831108628668                                                  
CTYPE7  = 'DEC     '                                                            
CRVAL7  =  -19.551308566410768 
```

A larger difference is the array XYZ location, which MS has at:

```
ARRAYX  =    -2559453.29059553                                                  
ARRAYY  =     5095371.73544116                                                  
ARRAYZ  =    -2849056.77357178   
```

whereas the aavs_uv has at:

```
ARRAYX  =    -2559377.99363595                                                  
ARRAYY  =     5095349.74793982                                                  
ARRAYZ  =    -2849124.06772244   
```

which matches the XYZ position calculated on the [MCCS-1826](https://confluence.skatelescope.org/display/SE/MCCS-1826%3A+Provide+AAVS3+data+for+telmodel+export) confluence page. 

Plotting the UV coordinates, data look to be in about as much agreement as I would expect, given the differences of XYZ and phase center. 

![image](https://github.com/ska-sci-ops/aavs_uv/assets/713251/155766f3-893c-44ec-9436-157d57863737)

![image](https://github.com/ska-sci-ops/aavs_uv/assets/713251/3012c77f-70c5-4521-b1ac-30f223154457)

TL;DR: Main discrepancy was probably just not phasing to sun.
